doc/fitting/livefit_logger.md

```diff
@@ -0,0 +1,35 @@
+# Fitting Files Callback
+## Fitting Files
+
+The callback (`LiveFitLogger`) exists to write all fitting metrics from `LiveFit` to file. These are designed to be human readable files rather than machine readable.
```

Comment:
You can link this more nicely to API reference using something like 
```
{py:obj}`ibex_bluesky_core.blah.LiveFitLogger`
```

Which will then click through nicely to API docs etc